Package.swift

```diff
@@ -38,6 +38,7 @@ let package = Package(
             targets: [
                 "SourceControl",
                 "SPMLLBuild",
+                "PackageCollectionJSONModel",
```

Comment:
I find the name a bit overly descriptive and think just `PackageCollectionsModel` would be better.

Reply:
Amended https://github.com/apple/swift-package-manager/pull/3199/commits/fcf50914ecb0c566dfe77af696a99042a22e07ac